lib/cadet/assessments/assessments.ex

```diff
@@ -15,8 +15,9 @@ defmodule Cadet.Assessments do
   @xp_early_submission_max_bonus 100
   @xp_bonus_assessment_type ~w(mission sidequest)a
   @submit_answer_roles ~w(student)a
-  @grading_role :staff
-  @see_all_submissions_role :admin
+  @grading_roles ~w()a
+  @admin_role :admin
```

Comment:
Not sure what `@admin_role` is used for

Reply:
It was used under the `role != @admin_role` (refer to PR comment above)

I am not sure if it is really needed actually?